apps/04-golang-pgsql/Dockerfile

```diff
@@ -1,7 +1,4 @@
-ARG GO_BASE_IMAGE=golang:1.19-alpine
-ARG RUNTIME_BASE_IMAGE=alpine
-
-FROM ${GO_BASE_IMAGE} as build
```

Comment:
I hope it wasn't casing any issues?
This was done so that the base image can be switched to a ECR image or GCR image to avoid rate limit issues in cloud. While also supporting the DockerHub base by default for simplicity.
If you are fine with it, maybe you can add a comment here for future reference.